config/environments/production.rb

```diff
@@ -17,26 +17,17 @@
   config.i18n.fallbacks = true
 
   config.active_support.deprecation = :notify
-<<<<<<< HEAD
+
+  config.action_mailer.default_url_options = { host: 'myflixapp.herokuapp.com'}
 
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = {
-    address:              "smtp.gmail.com",
-    port:                 587,
-    user_name:            ENV['gmail_username'],
-    password:             ENV['gmail_password'],
-    authentication:       "plain",
-    enable_starttls_auto: true
-=======
-  config.action_mailer.delivery_method = :smtp
-  # SMTP settings for gmail
-  config.action_mailer.smtp_settings = {
-   :address              => "smtp.gmail.com",
-   :port                 => 587,
-   :user_name            => ENV['gmail_username'],
-   :password             => ENV['gmail_password'],
-   :authentication       => "plain",
-  :enable_starttls_auto => true 
->>>>>>> master
+    :port           => ENV['MAILGUN_SMTP_PORT'],
+    :address        => ENV['MAILGUN_SMTP_SERVER'],
+    :user_name      => ENV['MAILGUN_SMTP_LOGIN'],
+    :password       => ENV['MAILGUN_SMTP_PASSWORD'],
+    :domain         => 'myflixapp.heroku.com',
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.